ospo-book/chapters/01-chapter.md

```diff
@@ -18,3 +24,113 @@ The book is structured in a user-friendly and practical manner, with a focus on
+
+* Open Source Environmentalist: OSPOs can help organizations support and sustain open source projects in the long term by addressing issues such as security, maintenance, and project health. This can involve establishing policies and procedures for code review, security vulnerability management, and ongoing maintenance and support through funding and/or contributions. By doing so, OSPOs can help ensure that open source projects remain healthy and continue to benefit the wider community.
+
+## History and Roots
+
+The OSPO concept initially started within the corporate world and is now about two decades old, but really started to accelerate in the last decade or so. Most prominent technology infrastructure firms (e.g., Amazon, VMware, Cisco) and consumer technology companies (e.g., Apple, Google, Facebook) ended up having an OSPOs or formal open source programs. All are encouraging their employees to contribute to open source projects that are strategic to their business and security.
+
+The term started becoming more mainstream and diverse in the last years, as more organizations from different sectors and regions included dedicated open source roles in their organization to manage open source operations and strategy. Nowadays, we can find OSPOs being formed in different regions (APAC, EMEA, AMER) and entities, such as Governments, Enterprises, NGOs, Universities and more.
+
+> Important Considerations in Defining an Open Source Program Office (OSPO): OSPOs vary in Sector, Region, and Organizational Size; May Exclude the Term 'Program' to become 'Open Source Office'; and No Two OSPOs are Alike.
+
+# Assessing Readiness for Open Source and OSPO
+
+`✅ Assesment`
+
+The purpose of this section is to first identify the strengths, weaknesses, and opportunities for improvement within the organization, 
+and to help determine if an OSPO is the right solution for the organization's needs based on their existing open source engagement level, culture and understanding. 
+
+## Where do Open Source and OSPO converge?
+
+In the past, open source software was primarily used by small groups of developers and enthusiasts, and there was little need for dedicated organizational units to manage open source activities. However, as open source software has become more prevalent and critical to the operation of many organizations, the need for dedicated OSPOs has become more apparent.
+
+### Understand existing and desired open source adoption
+
+Once an organization has assessed the level of open source used, contributed, or produced in the organization and why establishing an OSPO can help an organization manage the risks and opportunities of what open source, open works and collaboration brings, and ensure that its open source activities are effectively managed and aligned with the organization's strategic goals and objectives. 
```

Comment:
What is open work?

Reply:
This is a very long sentence and I don't understand what it is intended to say.

Reply:
@CsatariGergely this link provides a clear definition 👍 http://openworkdefinition.com/ 

The reason for why including this term is because usually non-corporate OSPOs, focus not only on open source software, but also on other "works" topics such as artwork, open data, hardware, and more.